api/server.js

```diff
@@ -92,6 +92,32 @@ router.post('/api/events/createEvent', (req, res) => {
         });
 });
 
+// PUT -- update an event event
+router.put('/api/events/updateEvent', (req, res) => {
+    global.connection.query('UPDATE BetterLinkedIn_sp20.PlannedEvents SET (EventName, EventTime, EventDescription, IndustryID, OrganizerID) VALUES (?, ?, ?, ?, ?)',
+        [req.body.eventName, new Date(req.body.eventTime), req.body.eventDesc, req.body.industryID, req.body.userID],
+        (error, results, fields) => {
+            if (error) {
+                res.send(JSON.stringify({ status: 400, error, response: results }));
+                console.log(JSON.stringify({ status: 400, error, response: results }));
+            // If event is successfully created, create a record in Attending for the user as the event organizer
+            } else {
+                global.connection.query('UPDATE BetterLinkedIn_sp20.Attending SET (PersonID, EventID, IsOrganizer, RSVPDate) VALUES (?, ?, ?, ?)',
+                    [req.body.userID, results.insertId, 1, new Date()],
```

Comment:
Same with this, I feel like we should leave the organizer ID always the same because it's the creator of the event.

Reply:
I took out this else block because the attending list should not change after editing an event.